docsrc/content/type-nonempty.fsx

```diff
@@ -4,14 +4,44 @@
 #I "../../bin"
 
 (**
-TO-DO Add some docs here !
+NonEmptyList
 =========================
 
+A type-safe list that contains at least one element.
+
 Examples
 --------
 *)
 
 
 #r @"../../src/FSharpPlus/bin/Release/net45/FSharpPlus.dll"
 
-open FSharpPlus
\ No newline at end of file
+open FSharpPlus
+open FSharpPlus.Data
+
+(**
+### Constructing NonEmptyList
+*)
+// you can construct a NonEmptyList by using ofSeq
+let list123 = NonEmptyList.create 1 [ 2; 3 ]
```

Comment:
I think it worths mentioning the nel builder as well.
To me it looks even more idiomatic that create, when creating as literal expression, ie `nel {1; 2; 3}`